### ISSUE TITLE
Initial Model -2 (Improve the validation accuracy)

### ISSUE THREAD

**Issue by hrshtsharma17** [hrshtsharma17/ToothdecayNN#2]
Adding Dropout layer reduced the validation accuracy but also helped suppress the overfitting cause.
